### DISCUSSION TITLE
Chromium 加入 Google 同步（文章中链接更新）

### DISCUSSION THREAD

**Discussion by gua-leopard** [FreeBSD-Ask/FreeBSD-Ask#867] [Ideas]
https://gist.github.com/cvan/44a6d60457b20133191bd7b104f9dcc4 这个网页中的内容已经无效了，新的有效文章在https://www.learningtopi.com/sbc/chromium-sync/

**Comment by ykla** [comment]
根据群里的人反映虽然被划掉了，但是还可以用。具体我没有测过。

**Comment by ykla** [comment]
如果你没有测过的话，最近一个月内我会花时间分别测一下。

**Comment by gua-leopard** [comment]
好的

**Comment by ykla** [comment]
抱歉，我在短期内无法完成该测试。

**Comment by Dedicatus5457** [comment]
接 #131 ，我和 @ykla 的讨论，还是跟进到这个issue吧，不然别的地方太off topic了

![Image_1697112222589.jpg](https://github.com/FreeBSD-Ask/FreeBSD-Ask/assets/78254776/41709a63-e5aa-4737-9770-15a3849f31a4)

![IMG_20231012_200251.jpg](https://github.com/FreeBSD-Ask/FreeBSD-Ask/assets/78254776/c3b34b50-d705-4db3-94ab-e04376fe707b)

按照https://www.learningtopi.com/sbc/chromium-sync/
确实是有用的，我会抽空写篇文档发个pr，把上面这位发的原作者的文章带图翻译一遍加上把环境变量配到xprofile shell的rc这块的配置说一遍。

**Comment by ykla** [comment]
@Dedicatus5457 有任何进度?

**Comment by Dedicatus5457** [comment]
![image](https://github.com/FreeBSD-Ask/FreeBSD-Ask/assets/78254776/96b46093-7d0a-4cbd-8896-6133526a2eb7)


> @Dedicatus5457 有任何进度?

写了一半
https://github.com/Dedicatus5457/FreeBSD-Ask/commits/dedicatus5457-patch-02

**Comment by ykla** [comment]
@Dedicatus5457 遇到了问题了吗？

**Comment by ykla** [comment]
@Dedicatus5457 此部分内容相对重要，你是否还有意愿继续撰写。若 3 月内无回复将再次关闭。

**Comment by Dedicatus5457** [comment]
抱歉啊，之前那段时间状态不好没继续写完，今年过年这段时间前后肯定是有空写的，只是我这边实体机装freebsd的那台机器貌似有点硬件上的问题。可能最近只能用虚拟机测试了，不过对于这篇来讲效果都一样的。

祝好；
Warm Regards
Qiyao Hou
2025.1.7

发送自 Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by ykla** [comment]
@Dedicatus5457 请看一下在两年后的今天有无任何更便捷的方法。虚拟机没有什么影响。如果可以能补充一下代理的使用方法吗？我倾向于使用图形化的代理软件，比如 clash for Windows。

**Comment by Dedicatus5457** [comment]
> [@Dedicatus5457](https://github.com/Dedicatus5457) 请看一下在两年后的今天有无任何更便捷的方法。虚拟机没有什么影响。如果可以能补充一下代理的使用方法吗？我倾向于使用图形化的代理软件，比如 clash for Windows。

有的，[V2rayA](https://v2raya.org/)这个v2ray图形化客户端项目有FreeBSD的二进制包

**Comment by Dedicatus5457** [comment]
> > [@Dedicatus5457](https://github.com/Dedicatus5457) 请看一下在两年后的今天有无任何更便捷的方法。虚拟机没有什么影响。如果可以能补充一下代理的使用方法吗？我倾向于使用图形化的代理软件，比如 clash for Windows。
> 
> 有的，[V2rayA](https://v2raya.org/)这个v2ray图形化客户端项目有FreeBSD的二进制包

- https://github.com/v2rayA/v2rayA/releases/download/v2.2.6.3/v2raya_freebsd_x64_2.2.6.3

但Clash目前的主流桌面端GUI实现是 Clash Verge Rev ，基于Tauri，理论上可以将它打包为appimage用于FreeBSD平台？

- https://github.com/clash-verge-rev/clash-verge-rev

不过，appimage也需要debootstrap才能在FreeBSD上跑来着..

**Comment by ykla** [comment]
> > > [@Dedicatus5457](https://github.com/Dedicatus5457) 请看一下在两年后的今天有无任何更便捷的方法。虚拟机没有什么影响。如果可以能补充一下代理的使用方法吗？我倾向于使用图形化的代理软件，比如 clash for Windows。
> > 
> > 有的，[V2rayA](https://v2raya.org/)这个v2ray图形化客户端项目有FreeBSD的二进制包
> 
> - https://github.com/v2rayA/v2rayA/releases/download/v2.2.6.3/v2raya_freebsd_x64_2.2.6.3
> 
> 但Clash目前的主流桌面端GUI实现是 Clash Verge Rev ，基于Tauri，理论上可以将它打包为appimage用于FreeBSD平台？
> 
> - https://github.com/clash-verge-rev/clash-verge-rev
> 
> 不过，appimage也需要debootstrap才能在FreeBSD上跑来着..

只用 arch 兼容层做一些示例即可。应该有打包的吧。麻烦你写一下 clash 部分的，我主要用这个。

**Comment by Dedicatus5457** [comment]
> > > > [@Dedicatus5457](https://github.com/Dedicatus5457) 请看一下在两年后的今天有无任何更便捷的方法。虚拟机没有什么影响。如果可以能补充一下代理的使用方法吗？我倾向于使用图形化的代理软件，比如 clash for Windows。
> > > 
> > > 
> > > 有的，[V2rayA](https://v2raya.org/)这个v2ray图形化客户端项目有FreeBSD的二进制包
> > 
> > 
> > 
> > * https://github.com/v2rayA/v2rayA/releases/download/v2.2.6.3/v2raya_freebsd_x64_2.2.6.3
> > 
> > 但Clash目前的主流桌面端GUI实现是 Clash Verge Rev ，基于Tauri，理论上可以将它打包为appimage用于FreeBSD平台？
> > 
> > * https://github.com/clash-verge-rev/clash-verge-rev
> > 
> > 不过，appimage也需要debootstrap才能在FreeBSD上跑来着..
> 
> 只用 arch 兼容层做一些示例即可。应该有打包的吧。麻烦你写一下 clash 部分的，我主要用这个。

具体指的是 ‘archlinux-pacman’ 兼容层吗？ 好的，我近日用虚拟机测试一下

**Comment by ykla** [comment]
> > > > > [@Dedicatus5457](https://github.com/Dedicatus5457) 请看一下在两年后的今天有无任何更便捷的方法。虚拟机没有什么影响。如果可以能补充一下代理的使用方法吗？我倾向于使用图形化的代理软件，比如 clash for Windows。
> > > > 
> > > > 
> > > > 有的，[V2rayA](https://v2raya.org/)这个v2ray图形化客户端项目有FreeBSD的二进制包
> > > 
> > > 
> > > 
> > > * https://github.com/v2rayA/v2rayA/releases/download/v2.2.6.3/v2raya_freebsd_x64_2.2.6.3
> > > 
> > > 但Clash目前的主流桌面端GUI实现是 Clash Verge Rev ，基于Tauri，理论上可以将它打包为appimage用于FreeBSD平台？
> > > 
> > > * https://github.com/clash-verge-rev/clash-verge-rev
> > > 
> > > 不过，appimage也需要debootstrap才能在FreeBSD上跑来着..
> > 
> > 只用 arch 兼容层做一些示例即可。应该有打包的吧。麻烦你写一下 clash 部分的，我主要用这个。
> 
> 具体指的是 ‘archlinux-pacman’ 兼容层吗？ 好的，我近日用虚拟机测试一下

第 21.4 节 Linux 兼容层——基于 ArchLinux bootstrap

**Comment by ykla** [comment]
@Dedicatus5457 我拉你进团队，你看下你邮箱

**Comment by Dedicatus5457** [comment]
> [@Dedicatus5457](https://github.com/Dedicatus5457) 我拉你进团队，你看下你邮箱

收到，我尽量在近期完成。另外打算完善润色一下README ，.gitconfig那块没有写gitconfig的作用域和repo里的默认path这些细节，稍后完善一下

**Reply by ykla** [reply]
此 todo（代理软件 GUI）仍有待完成，故迁移到了这里。
